ova/README.md

```diff
@@ -3,37 +3,10 @@ Wazuh Virtual Machine
 
 In this repository, you can find the necessary tools to build your own OVA file with all Wazuh components installed.
 
-## Tools needed to build the OVA:
-
-To build an OVA you need to install the following tools:
-- `Virtual Box`: [installation guide](https://www.virtualbox.org/manual/UserManual.html#installation)
-- `Vagrant`: [installation guide](https://www.vagrantup.com/docs/installation/)
-- `Git`:  [installation guide](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
-- `Python`: [installation guide](https://www.python.org/download/releases/2.7/)
-
 
 ## Building OVA file:
 
-To generate the ova file, you need to download this repository and use the `generate_ova.sh` script. This script will create a vagrant box and provision it to be ready to use wazuh, as specified on provision.sh, and then export it to an Ova file.
-
-```shellsession
-# ./generate_ova.sh -h
-
-OPTIONS:
-      -b, --build            [Required] Build the OVA and OVF.
-      -v, --version          [Required] Version of wazuh to install on VM.
-      -e, --elastic-version  [Required] Elastic version to download inside VM.
-      -r, --repository       [Required] Status of the packages [stable/unstable]
-      -c, --clean            [Optional] Clean the local machine.
-      -h, --help             [  Util  ] Show this help.
-```
-
-To build an OVA with version 3.9.5 using elastic 7.3.0 and the stable repositories you can use:
-
-`# ./generate_ova.sh -b -v 3.9.5 -e 7.3.0 -r stable`
-
-   * **Stable:** The OVA uses released packages.
-   * **Unstable:** The OVA uses unstable packages.
+All info is in [Generate-ova.](https://documentation.wazuh.com/3.11/packaging/generate-ova.html)
```

Comment:
Please, replace "All info is in" by "Please, visit the following link for the full OVA building documentation:"

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/349/commits/90b419309b49288b9216ca08c55d61243f1e6bae.